.github/workflows/unit-tests.yml

```diff
@@ -31,52 +34,128 @@ jobs:
           flags: javascript
 
   python-unit:
-    name: python unit tests
+    name: Python unit tests
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+         # for python 3.5, only test with mariadb and solr 6
+         - python: 3.6
+           solr: 6.6.5
+           db: mysql
+        # for python 3.6, only test with postgres and solr 8
+         - python: 3.6
+           solr: 8.6
+           db: postgresql
     services:
-      db:
+      mariadb:
         image: mariadb
         env:
-          MYSQL_DATABASE: ppa
-          MYSQL_ROOT_PASSWORD: ppa
+          MYSQL_DATABASE: ${{ env.DB_NAME }}
+          MYSQL_ROOT_PASSWORD: ${{ env.DB_PASSWORD }}
         ports:
-          - 3306:3306
+        - 3306:3306
+      postgres:
+        image: postgres:12
+        env:
+          POSTGRES_DB: ${{ env.DB_NAME }}
+          POSTGRES_USER: ${{ env.DB_USER }}
+          POSTGRES_PASSWORD: ${{ env.DB_PASSWORD }}
+        ports:
+          - 5432:5432
       solr:
-        image: solr:6.6
+        image: solr:${{ matrix.solr }}
         ports:
           - 8983:8983
     steps:
-      - uses: actions/checkout@v2
-      - run: docker cp solr_conf ${{ job.services.solr.id }}:/opt/solr/server/solr/configsets/ppa
-      - run: docker exec --user root ${{ job.services.solr.id }} /bin/bash -c "chown -R solr:solr /opt/solr/server/solr/configsets/ppa"
-      - run: echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
-      - uses: actions/setup-python@v2
+      # Set the value of DJANGO_DB_BACKEND which is used in ci/testsettings.py to
+      # configure django's ORM based on whether we're testing postgres or mysql
+      - name: Set django database backend adapter
+        env:
+          BACKEND: ${{ matrix.db }}
+        run: echo "DJANGO_DB_BACKEND=$(echo "$BACKEND")" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # use docker cp to copy the configset, then bash to set ownership to solr
+      - name: Copy Solr configset to solr service
+        run: |
+          docker cp solr_conf ${{ job.services.solr.id }}:/opt/solr/server/solr/configsets/ppa
+          docker exec --user root ${{ job.services.solr.id }} /bin/bash -c "chown -R solr:solr /opt/solr/server/solr/configsets/ppa"
+
+      - name: Copy solr configsets to solr home directory (Solr 8 only)
+        run: "docker exec -d ${{ job.services.solr.id }} cp -r /opt/solr/server/solr/configsets /var/solr/data"
+        if: ${{ matrix.solr == 8.6 }}
+
+      # NOTE: for duration of migration to PUL, use python version from build matrix
+      # Python version to use is stored in the .python-version file, which is the
+      # convention for pyenv: https://github.com/pyenv/pyenv
+      # - name: Get Python version
+      #   run: echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
```

Comment:
we aren't changing the python version, are we? same for both? I would use it here and not in the matrix, but may not matter so much

Reply:
I believe we initially were -- I should have noted this -- using Python 3.5 for the mysql + solr6 configuration, but I was getting an error that stackoverflow suggested was an artifact of 3.5 vs 3.6, so I switched to 3.6 for the time being. Let me know what you want to do – I can try to get 3.5 to work tho I also think 3.6 is reasonable to expect at this point?

Reply:
oh! my apologies, that's another leftover from the mep code I pointed you to — in that case we were still on 3.5 but migrating to 3.6, in ppa we're already on 3.6. Let's restore this bit and remove python version from the matrix